Dockerfile

```diff
@@ -11,7 +11,7 @@ RUN mkdir /proxy &&\
 
 # Fetch goose
 RUN export GO111MODULE=on &&\
-    go get -v -u github.com/pressly/goose/cmd/goose@v2.7.0-rc5
```

Comment:
I had to change this since one of the dependencies of this module is not compatible with golang:1.15. And since I updated Golang version - I decided to also update goose version.